a-deux-pas/front-a-deux-pas
# Feature/dockerfile  Docker Configuration Update

This PR introduces the following changes:

1. New Docker image:
   - Updated base image
   - Optimized layers for better performance

2. Nginx configuration:
   - Added custom Nginx configuration file
   - Optimized settings for serving the Angular application

3. Added .dockerignore file:
   - Excluded unnecessary files and folders during Docker build
   - Improved build performance and reduced image size

These changes aim to enhance the performance, security, and maintainability of our Docker environment.

## Testing
- Build the Docker image : docker build -t my-angular-app .
- Start the container and verify the application's functionality:  docker run -d -p 80:80 --name my-angular-container my-angular-app
- Check Nginx configuration (routes, performance) : docker run -d -p 80:80 --name my-angular-container my-angular-app
